packages/devtools_app/test/integration_tests/integration.dart

```diff
@@ -16,40 +14,40 @@ import 'package:webkit_inspection_protocol/webkit_inspection_protocol.dart'
 
 const bool verboseTesting = false;
 
-WebBuildFixture webBuildFixture;
-BrowserManager browserManager;
+late WebBuildFixture webBuildFixture;
+late BrowserManager browserManager;
 
 class DevtoolsManager {
   DevtoolsManager(this.tabInstance, this.baseUri);
 
-  final BrowserTabInstance tabInstance;
+  final BrowserTabInstance? tabInstance;
   final Uri baseUri;
 
   Future<void> start(
     AppFixture appFixture, {
-    Uri overrideUri,
+    Uri? overrideUri,
     bool waitForConnection = true,
   }) async {
     final Uri baseAppUri = baseUri.resolve(
         'index.html?uri=${Uri.encodeQueryComponent(appFixture.serviceUri.toString())}');
-    await tabInstance.tab.navigate('${overrideUri ?? baseAppUri}');
+    await tabInstance!.tab.navigate('${overrideUri ?? baseAppUri}');
 
     // wait for app initialization
     await Future.wait([
       waitForConnection
-          ? tabInstance.onEvent
+          ? tabInstance!.onEvent
               .firstWhere((msg) => msg.event == 'app.devToolsReady')
           : Future.value(),
-      tabInstance.getBrowserChannel(),
+      tabInstance!.getBrowserChannel(),
     ]);
   }
 
   Future<void> switchPage(String page) async {
-    await tabInstance.send('switchPage', page);
+    await tabInstance!.send('switchPage', page);
   }
 
-  Future<String> currentPageId() async {
-    final AppResponse response = await tabInstance.send('currentPageId');
+  Future<String?> currentPageId() async {
```

Comment:
The other methods that return `response.result` are cast as `String?` - does this return need that also? if not, do the other casts actually need to be there?

Reply:
Not sure why we do it differently. Added casting here and in other places for consistency.